webauthn-rs-core/src/core.rs

```diff
@@ -1532,6 +1567,7 @@ mod tests {
             registration_policy: UserVerificationPolicy::Discouraged_DO_NOT_USE,
             extensions: RegisteredExtensions::none(),
             attestation: ParsedAttestationData::None,
+            attestation_format: None,
```

Comment:
We don't need an attestation_format flag, because the parsesAttestationData enum indicates the format that wraps it doesn't it?

Reply:
Ahhh as I saw above, it looks like these use "Basic" in parsedAttestationData.